app/views/register-your-interest/registration-complete.njk

```diff
@@ -0,0 +1,51 @@
+{% extends './layouts/layout.njk' %}
+
+{% block pageTitle %}{{serviceName}}{% endblock %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: "Back",
```

Comment:
Can remove the back button from this page

Reply:
good catch!